lib/helper/desktopHelper.py

```diff
@@ -36,32 +36,24 @@ def launch_browser(browser_type, **kwargs):
 
     chrome_class, firefox_class = _load_browser_class(engine_type)
 
-    if env.PROFILER_FLAG_CHROMETRACING in enabled_profiler_list:
-        if browser_type == env.DEFAULT_BROWSER_TYPE_CHROME:
+    if browser_type == env.DEFAULT_BROWSER_TYPE_CHROME:
+        if env.PROFILER_FLAG_CHROMETRACING in enabled_profiler_list:
             profile_path = env.chrome_profile_path
             browser_obj = chrome_class(env.DEFAULT_BROWSER_HEIGHT, env.DEFAULT_BROWSER_WIDTH,
                                        tracing_path=env.chrome_tracing_file_fp,
                                        profile_path=profile_path)
         else:
-            profile_path = env.firefox_profile_path
-            browser_obj = firefox_class(env.DEFAULT_BROWSER_HEIGHT, env.DEFAULT_BROWSER_WIDTH,
-                                        profile_path=profile_path)
-    elif env.PROFILER_FLAG_FXTRACELOGGER in enabled_profiler_list:
-        if browser_type == env.DEFAULT_BROWSER_TYPE_FIREFOX:
+            profile_path = env.chrome_profile_path
```

Comment:
the `profile_path = env.firefox_profile_path` for Chrome at L.41 and L.46 are the same, we can extract them to L.40.
ex:
```diff
    if browser_type == env.DEFAULT_BROWSER_TYPE_CHROME:
+        profile_path = env.chrome_profile_path
        if env.PROFILER_FLAG_CHROMETRACING in enabled_profiler_list:
-            profile_path = env.chrome_profile_path
            browser_obj = chrome_class(env.DEFAULT_BROWSER_HEIGHT, env.DEFAULT_BROWSER_WIDTH,
                                       tracing_path=env.chrome_tracing_file_fp,
                                       profile_path=profile_path)
        else:
-            profile_path = env.chrome_profile_path
            browser_obj = chrome_class(env.DEFAULT_BROWSER_HEIGHT, env.DEFAULT_BROWSER_WIDTH, profile_path=profile_path)
```